aamt.pl

```diff
@@ -0,0 +1,454 @@
+use strict;
```

Comment:
aamt.pl should be deleted and the changes should be applied to migrate_tool_main.pl